### PR TITLE
Add test for bug #75140

### PR DESCRIPTION
**Pull request by johnkary**
Contributing as part of PHP Test Fest for Kansas City PHP User Group
🎉 

[Bug #75140](https://bugs.php.net/bug.php?id=75140) is related to [`session.sid_length`](http://php.net/manual/en/session.configuration.php#ini.session.sid-length) which was introduced in PHP 7.1.0, so this PR is against 7.1 branch.

If format of this commit or the PHPT file should be different let me know!

Thanks,
John Kary
Co-Organizer, Kansas City PHP User Group

### CODE REVIEW

**Comment by ericpoe** [comment]
Can someone kick the tires of this? It looks like travis-ci had a hiccup while running the non-debug version of the build.

**Comment by ramsey** [comment]
I just restarted that build. Hopefully, it completes properly this time.

File: ext/session/tests/bug75140.phpt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+--INI--
+session.sid_length=251
+--FILE--
+<?php
+
+session_start();
+echo "ok\n";
+
+?>
+--XFAIL--
```
Comment (carusogabriel) [inline ext/session/tests/bug75140.phpt:14]: We won't accept `XFAIL` tests :confused:

**Review by carusogabriel — CHANGES_REQUESTED** [review]

**Comment by petk** [comment]
Hello, thank you for the pull request. This has been also opened at https://github.com/php/php-src/pull/3802 and it seems that the only proper way here is to fix the mentioned bug instead. XFAIL tests aren't being added in the php-src anymore unfortunately.

**Pull request closed without merging**